test/integration/page-objects/catalog.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const h = require('../helpers.js');
+const inputs = require('../helpers/inputs');
+const Page = require('./page').Page;
+
+let AddTemplateModal = function(project) {
+  this.project = project;
+  this.modal = element(by.css('.modal-dialog'));
+  this.checkboxes = this.modal.all(by.css('input[type="checkbox"]'));
+  this.processBox = this.checkboxes.get(0);
```

Comment:
Agree, let me look into that, i dont remember if we have an id or class on the input.
- [ ] find a better selector not based on order (id, etc)